src/cryptonote_basic/verification_context.h

```diff
@@ -83,7 +83,7 @@ namespace cryptonote
     bool m_invalid_type;
     bool m_key_image_locked_by_snode;
     bool m_key_image_blacklisted;
-
+    std::string m_verbose_error;
```

Comment:
This is going to explode the size of a `tx_verification_context` and make it non-trivially copyable, where it's probably being copied like an int currently.  But that said, I don't really have a better solution; this is kind of a poster child for why exceptions were invented.